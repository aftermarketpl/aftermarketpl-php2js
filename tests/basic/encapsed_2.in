$a = array("a");
return "b{$a[0]}";
