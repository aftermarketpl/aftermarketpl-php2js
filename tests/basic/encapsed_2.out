a = [
    "a"
];
return ("b" + a[0]);
